File: workbench/JS/test2.js
 const prompt = require('prompt-sync')();

function reverse(n) {
    
   let nr = n.length-1
    let s = "" 

        
    for(let i = nr; i >= 0; i-- ){
        s += n[i]
    }
    
    return s;

}

let s = "hola"
console.log("string reversado: " + reverse(s))

File: workbench/Go/test2.go
package main

import (
	"fmt"
)

func reverse(s string) string {

	n := len(s) - 1

	str := " "
	for i := n; i > 0; i-- {
		str[i] = str + s[i]
	}
	return str
}

func main() {

	a := "Hello"

	fmt.Println(reverse(a))

	fmt.Println(len(a) - 1)

}
